components/pages/home/Navbar.js

```diff
@@ -0,0 +1,191 @@
+import Image from "next/image";
+import { Icon } from "@iconify/react";
+import Link from "next/link";
+import { useState } from "react";
```

Comment:
Posisi penulisan importnya disusun ulang biar enak dibacanya

```suggestion
import { useState } from "react";
import Link from "next/link";
import Image from "next/image";
import { Icon } from "@iconify/react";
```